### PR TITLE
Feat/redesign event detail page

### PR DESCRIPTION
**Pull request by deannabosschert**
## Description
- Added better/more styling to the event detail page
- Added more content
- Fixed current content
- Fixed rendering of said content
- Added filter to parse Rich Text with
- Added filter to map+parse the 'Section text' component with
- Made some changes that makes the files correspond with the current master branch which avoids my PR having any pesky merge conflicts

## Testing
Go to `http://localhost:3000/events/golden-dot-awards-2020/`.

Does it look like this?:
screenshot of Golden Dot Awards detailpage
plus, check if all content is rendered correctly (so; no [Object Object] stuff or something)

## To do after merging this PR (optional)
If you feel like it, re-arrange the content and/or change some breakpoints. I'm not a visual designer so it isn't pretty, but could've been worse. I do like some of the details 🤷

### CODE REVIEW

File: lib/filters/map-section-text.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const parseRichText = require('./parse-rich-text')
+
+module.exports = (sectionTexts) => {
+  const mappedSectionTexts = sectionTexts.map(section => {
```
[the remark this answers is not in the document]
Comment (kriskuiper) [reply lib/filters/map-section-text.js:4]: Zoiets kan je ook doen om de error te voorkomen:

```js
const mappedSectionTexts = sectionTexts && sectionTexts.map({ ... })
```

File: src/site/_includes/components/events/details/event-details.scss [hunk 2]
```diff
@@ -1,21 +1,330 @@
...
+
+.event-details__contactForm__form {
+  padding: $spacing-small $spacing-small $spacing-default 0;
+}
+
+.event-details__contactForm__form .app-form__input-group {
+  max-width: 80%;
+}
+
+// ANIMATIONS- bubbles
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/site/_includes/components/events/details/event-details.scss]: Done!

File: src/site/_includes/components/events/details/event-details.scss [hunk 3]
```diff
@@ -1,21 +1,330 @@
...
+@media (min-width: $bp-small) {
+  .event-details__contactForm,
+  .event-details__information__section,
+  .event-details__introduction__description,
+  .event-details__introduction__meta {
+    min-width: 80vw;
+    max-width: 90vw;
+  }
+}
+@media (min-width: $bp-medium) {
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/site/_includes/components/events/details/event-details.scss:125]: Done

File: src/site/_includes/components/events/details/event-details.scss [hunk 4]
```diff
@@ -1,21 +1,330 @@
...
+  .event-details__contactForm,
+  .event-details__information__section,
+  .event-details__introduction__description,
+  .event-details__introduction__meta {
+    min-width: 60vw;
+    max-width: 65vw;
+  }
+}
+@media (min-width: $bp-large) {
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/site/_includes/components/events/details/event-details.scss:135]: Done

File: src/site/_includes/components/events/details/event-details.scss [hunk 5]
```diff
@@ -1,21 +1,330 @@
...
+//SUBELEMENTS- information
+.event-details__information__section {
+  background: $color-white;
+  box-shadow: 6px 6px 9px $color-smokewhite, -6px -6px 9px $color-white;
+  border-radius: $border-radius-default;
+  padding: $spacing-medium;
+  margin-bottom: $spacing-huge;
+  margin-right: $spacing-large;
+}
+@media (min-width: $bp-small) {
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/site/_includes/components/events/details/event-details.scss:115]: Done

File: lib/get-events.js [hunk 6]
```diff
@@ -1,15 +1,25 @@
...
-          description: event.content.meta_description
-        },
-        full_slug: event.full_slug,
-        slug: event.slug
-      }
-    })
+  .map(event => {
+    const parsedDescription = parseRichText(event.content.description)
+    const sectionTexts = mapSectionText(event.content.section_text)
+    const contactForm_text = mapSectionText(event.content.contact_form)
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply lib/get-events.js:9]: Omdat het binnen het component 'contactForm' valt maar ik het als seperate entity wil kunnen accessen

File: lib/filters/parse-rich-text.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+const Storyblok = require('../../src/lib/storyblok-instance.js')
+
+module.exports = (richText) => {
+    const parsedText = Storyblok.richTextResolver.render(richText)
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply lib/filters/parse-rich-text.js]: Done!

File: lib/filters/map-section-text.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (deannabosschert) [reply lib/filters/map-section-text.js]: Done! Er staan nog twee `The render method must receive an object with a content field, which is an array`-errors in de Terminal maar ik kan zelf niet zien/checken in welke file de error zit.  Ik deploy 'm zo opnieuw, kun je dan de build logs checken?

**Pull request merged**